x11-packages/tilix/build.sh

```diff
@@ -0,0 +1,26 @@
+TERMUX_PKG_HOMEPAGE=https://gnunn1.github.io/tilix-web
+TERMUX_PKG_DESCRIPTION="A tiling terminal emulator for Linux using GTK+ 3"
+TERMUX_PKG_LICENSE="MPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_VERSION=1.9.6
+TERMUX_PKG_SRCURL=(https://github.com/gnunn1/tilix/archive/refs/tags/1.9.6.tar.gz
```

Comment:
this should be based on the `$TERMUX_PKG_VERSION`.
```suggestion
TERMUX_PKG_SRCURL=(
	"https://github.com/gnunn1/tilix/archive/refs/tags/${TERMUX_PKG_VERSION[0]}.tar.gz"
	"https://github.com/gtkd-developers/GtkD/archive/refs/tags/v${TERMUX_PKG_VERSION[1]}.tar.gz"
)
```